app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self, distance_from_city_center: float,
+            clean_power: int, average_rating: float,
+            count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            car.comfort_class * (self.clean_power - car.clean_mark)
+            * self.average_rating / self.distance_from_city_center, 1
+        )
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
+
+    def rate_service(self, single_rate: int) -> None:
+        sum_rates = self.average_rating * self.count_of_ratings + single_rate
+        self.count_of_ratings += 1
+        self.average_rating = round(sum_rates / self.count_of_ratings, 1)
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += round(
+                    car.comfort_class
+                    * (self.clean_power - car.clean_mark)
+                    * self.average_rating
+                    / self.distance_from_city_center,
+                    1
+                )
+                self.wash_single_car(car)
+        return income
```

Comment:
The `serve_cars` method duplicates the logic of calculating the washing price. Consider using the `calculate_washing_price` method within `serve_cars` to avoid code duplication and ensure consistency in price calculation.